ingestjob/logger.go

```diff
@@ -256,3 +257,51 @@ func generateID(log *Log) (string, error) {
 	}
 	return docID, nil
 }
+
+// WriteTask ...
+func (s *Logger) WriteTask(log *TaskLog) error {
+	if log.Connector == "" || len(log.Configuration) == 0 || log.CreatedAt.IsZero() {
+		return fmt.Errorf("error: log connector, configuration and created at are all required")
+	}
+
+	docID, err := generateTaskID(log)
+	if err != nil {
+		return err
+	}
+
+	b, err := jsoniter.Marshal(log)
+	if err != nil {
+		return err
+	}
+
+	index := fmt.Sprintf("%s-%s-log-%s", tasksIndex, log.Connector, s.environment)
```

Comment:
@aultron as per ticket requirements the index needs to be in the format `insights-tasks-{env}`.